orbit/pkg/packaging/wix/wix.go

```diff
@@ -39,14 +25,14 @@ func Heat(path string, native bool) error {
 		args = append(
 			args,
 			"docker", "run", "--rm", "--platform", "linux/amd64",
-			"--volume", path+":"+path, // mount volume
+			"--volume", path+":/wix", // mount volume
```

Comment:
this was the crux of the problem, as we were providing Windows-like paths on Windows for the bind mount